client/sites/settings/controller.tsx

```diff
@@ -43,10 +50,19 @@ export function siteSettings( context: PageJSContext, next: () => void ) {
 }
 
 export function administrationSettings( context: PageJSContext, next: () => void ) {
+	const state = context.store.getState();
+	const siteId = getSelectedSiteId( state );
+
 	context.primary = (
 		<PanelWithSidebar>
 			<SettingsSidebar />
-			<AdministrationSettings />
+			{ ! isSiteWpcomStaging( state, siteId ) ? (
+				<AdministrationSettings />
+			) : (
+				<SettingsNotSupportedNotice
+					message={ __( 'This setting is not supported for staging sites.' ) }
```

Comment:
Should this show a heading?

Reply:
I'm fine with not showing a heading to simplify things. It's really an edge case anyway.

Reply:
nit: we can also add a link to point to the original site. But yes, it's an edge case.